Feature: add the new composite page

ebuka-nwabueze opened the pull request.

ebuka-nwabueze committed:
add the new composite page

ebuka-nwabueze committed:
adjust the service card to be mobile responsive

replicate the composite with the orthopaedics

ebuka-nwabueze committed:
data: extract the data sections of composite and orthopaedics

ebuka-nwabueze committed:
rename composite page

ebuka-nwabueze committed:
change hamburger icon

ebuka-nwabueze committed:
change hamburger icon

ebuka-nwabueze committed:
increase gap for the infocardsection

ebuka-nwabueze committed:
extracted category into its own component

ebuka-nwabueze committed:
change the url for the categories link

Ozoemenamsolomon commented on src/pages/composites/_index.tsx:
```diff
@@ -0,0 +1,188 @@
+import calculator from '@assets/new/icons/calculate.svg';
```
remove the files that aren't needed

Ozoemenamsolomon replied:
or is this the new page? then rename it to the index without underscore

ebuka-nwabueze replied:
It's the one i mentionend to you earlier. i have removed it

ebuka-nwabueze committed:
remove old composite index page

Ozoemenamsolomon commented on src/pages/composites/categories/index.tsx:
```diff
@@ -1,12 +1,18 @@
-import Categories from '@components/Categories';
```
Did you check these removed codes for dead codes, like delete them if they aren't used anywhere else

Ozoemenamsolomon replied:
there are a lot of removed imports in the `pages/composites/index.tsx` file

ebuka-nwabueze replied:
What dead codes and imports are you referring to?

Ozoemenamsolomon replied:
[Dead codes](https://en.wikipedia.org/wiki/Dead_code): Codes that are not bein used, imported anywhere. For example [here](https://github.com/Ozoemenamsolomon/orthoex-store/pull/14/files#diff-c19bdcd7a2f8362a946e012e50fc63bd1701a4a73b57890b45ed1cd6ca217c13L18). Check if the `ViewMoreLink` is used or imported anywhere else, if not, delete the file.

ebuka-nwabueze replied:
Ok. I know what dead codes are, you weren't specific. 
 It's been removed.

Ozoemenamsolomon commented on src/pages/composites/categories/index.tsx:
```diff
@@ -24,8 +30,7 @@ const composite = () => {
 				style={{ display: 'flex', flexDirection: 'column' }}
 				header={{ title: 'All Categories', align: 'left' }}>
-				<Categories categories={categories} />
-				<ViewMoreLink href="/composites/products" text="View more Products" />
```
I personally think this could be reused because it has taken care of the icon. But you can delete it if you don't want to use it again

ebuka-nwabueze replied:
I think you may be looking at an older version of the file. I can't see what you are referring to

Ozoemenamsolomon replied:
I'm talking about line 28

ebuka-nwabueze committed:
remove the ViewMoreLink component

Ozoemenamsolomon committed:
Chores: Remove dead codes
- rename productVertical `title` to `name`
- organize imports
- remove unused imports

Ozoemenamsolomon reviewed (approved):
LGTM
